src/main/rx/rx.c

```diff
@@ -507,7 +507,7 @@ FAST_CODE_NOINLINE void rxFrameCheck(timeUs_t currentTimeUs, timeDelta_t current
         {
             const uint8_t frameStatus = rxRuntimeState.rcFrameStatusFn(&rxRuntimeState);
             DEBUG_SET(DEBUG_RX_SIGNAL_LOSS, 1, (frameStatus & RX_FRAME_FAILSAFE));
-            signalReceived = (frameStatus & RX_FRAME_COMPLETE) && !((frameStatus & RX_FRAME_FAILSAFE) || (frameStatus & RX_FRAME_DROPPED));
+            signalReceived = (frameStatus & RX_FRAME_COMPLETE) && !(frameStatus & (RX_FRAME_FAILSAFE | RX_FRAME_DROPPED));
```

Comment:
Is this intended to be a bitwise OR now, or should it be a logical OR (i.e. `||`)?

Reply:
Ahh I see... you are combining the bits for `RX_FRAME_FAILSAFE` and `RX_FRAME_DROPPED` first, and then doing the binary `&` against both at once 👍

Reply:
Thanks Wez!

Reply:
`RX_FRAME_FAILSAFE` and `RX_FRAME_DROPPED` are superimposed/overlayed. This superposition is then used as a filter to isolate the coresponding bits in `frameStatus`. If any of those two bits in `frameStatus` is 1, the resulting value is not equal 0, which evaluates to `true`. If both bits are 0, this oviously evaluates to `false`.